### DISCUSSION TITLE
availability of ESM2 MGnify embeddings

### DISCUSSION THREAD

**Discussion by y-hwang** [facebookresearch/esm#409] [ESM Metagenomic Atlas]
Hello, thanks for providing this excellent tool. I would like to know if your team will be making EMS2 protein embeddings available for the MGnify protein database. Thanks in advance.

**Comment by tomsercu** [comment]
We've had quite some folks request this and have it on our internal tracker for improving the ESM atlas, but can't commit to a timeline on this right now.

**Comment by chasemc** [comment]
Also interested
